spec/requests/admin/users_controller_spec.rb

```diff
@@ -67,6 +67,7 @@
     end
 
     it 'calls approve' do
+      StaffActionLogger.new(admin).expects(:log_user_approve).once
```

Comment:
It's better to check that the proper `UserHistory` was created rather than the `log_user_approve` method was called.